vue3_ts/src/views/veevalidate-yup/veevalidate-yup.enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum VALIDATION {
```

Comment:
please place it under src/views/veevalidate-yup/veevalidate-yup.types.ts
Enum has to be prefixed with 'E' letter and has PascalCase like EPascalCase.
Enum's properties has to be as UPPER_SNAKE_CASE

Reply:
@v-romanyuk I have small comment: not sure enum properties always should be: UPPER_SNAKE_CASE. IT's not convinient. For example our routeNames also should be enum, but it's not in snake_case. What do you think, can we do exception here?